backend/grants/admin.py

```diff
@@ -461,9 +459,6 @@ class GrantAdmin(ExportMixin, ConferencePermissionMixin, admin.ModelAdmin):
                     "total_amount",
                     "applicant_reply_sent_at",
                     "applicant_reply_deadline",
-                    "pretix_voucher_id",
```

Comment:
It might be beneficial to expose in this admin something that checks if a ConferenceVoucher exists for this grant user, so we don't need to go to a different UI to check if this grant has the voucher or not, what do you think?

Reply:
yeah was thinking about.

Reply:
I added the columns `has_voucher` and `has_ticket`